### ISSUE TITLE
from version 3.2 does not work (crashes) in vs 2017 on start

### ISSUE THREAD

**Issue by ark2git** [lkytal/SmoothScroll#30]
from version 3.2 does not work (crashes) in vs 2017.
on vs start shows message box saying:
 **"The SmoothScrollPackage did not load correctly.
The problem may have been caused by a configuration change or by the installation of another extension. You can get more information by examining the file 'C:\Users\*********\ActivityLog.xml'."**

From ActivityLog.xml:

```xml
<entry>
    <record>2637</record>
    <time>2020/06/07 10:25:46.436</time>
    <type>Error</type>
    <source>VisualStudio</source>
    <description>SetSite failed for package [SmoothScrollPackage]Source: &apos;SmoothScroll&apos; Description: Could not load file or assembly &apos;Microsoft.VisualStudio.Threading, Version=16.6.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a&apos; or one of its dependencies. The system cannot find the file specified.&#x000D;&#x000A;System.IO.FileNotFoundException: Could not load file or assembly &apos;Microsoft.VisualStudio.Threading, Version=16.6.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a&apos; or one of its dependencies. The system cannot find the file specified.&#x000D;&#x000A;File name: &apos;Microsoft.VisualStudio.Threading, Version=16.6.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a&apos;&#x000D;&#x000A;   at SmoothScroll.SmoothScrollPackage.InitializeAsync(CancellationToken cancellationToken, IProgress`1 progress)&#x000D;&#x000A;   at Microsoft.VisualStudio.Shell.AsyncPackage.&lt;&gt;c__DisplayClass17_0.&lt;&lt;Microsoft-VisualStudio-Shell-Interop-IAsyncLoadablePackageInitialize-Initialize&gt;b__1&gt;d.MoveNext()&#x000D;&#x000A;--- End of stack trace from previous location where exception was thrown ---&#x000D;&#x000A;   at Microsoft.VisualStudio.Services.VsTask.RethrowException(AggregateException e)&#x000D;&#x000A;   at Microsoft.VisualStudio.Services.VsTask.InternalGetResult(Boolean ignoreUIThreadCheck)&#x000D;&#x000A;   at Microsoft.VisualStudio.Services.VsTask.GetResult()&#x000D;&#x000A;&#x000D;&#x000A;WRN: Assembly binding logging is turned OFF.&#x000D;&#x000A;To enable assembly bind failure logging, set the registry value [HKLM\Software\Microsoft\Fusion!EnableLog] (DWORD) to 1.&#x000D;&#x000A;Note: There is some performance penalty associated with assembly bind failure logging.&#x000D;&#x000A;To turn this feature off, remove the registry value [HKLM\Software\Microsoft\Fusion!EnableLog].&#x000D;&#x000A;</description>
    <guid>{6BB22343-DF63-4F19-8088-66CAEAFDE0AD}</guid>
    <hr>80070002</hr>
    <errorinfo></errorinfo>
  </entry>
  <entry>
    <record>2638</record>
    <time>2020/06/07 10:25:46.436</time>
    <type>Error</type>
    <source>VisualStudio</source>
    <description>End package load [SmoothScrollPackage]</description>
    <guid>{6BB22343-DF63-4F19-8088-66CAEAFDE0AD}</guid>
    <hr>80070002</hr>
    <errorinfo></errorinfo>
  </entry>
```

**Comment by lkytal** [comment]
Should be fixed in 3.2.1, please feedback

**Comment by ark2git** [comment]
Confirmed - fixed, thanks.
I'm closing the issue.
